dev_environment/cfg/port_quota_add_payload.json

```diff
@@ -0,0 +1,25 @@
+[
+    {
+      "host_name": "l3af",
+      "iface": "Ethernet 4",
+      "bpf_programs": {
+        "xdp_ingress": [
+          {
+            "name": "port_quota",
+            "seq_id": 1,
+            "artifact": "port_quota.exe",
```

Comment:
Instead of port_quota, I would use rate-limiting since it is an xdp program, the same for the delete_payload. The name of the config file needs to change as well to something like "add_payload_win.json" and similarly for delete.  Please make sure all the config files uploaded here are tested and work seamlessly. Also, I would use "iface": "enp0s3" to be consistent with the other config files.